docs/data/base/guides/working-with-tailwind-css/working-with-tailwind-css.md

```diff
@@ -0,0 +1,503 @@
+# Working with Tailwind CSS
+
+<p class="description">Learn how to style MUI Base components with Tailwind CSS.</p>
+
+## Getting started
+
+The goal of this guide is to teach you how to style MUI Base components using Tailwind CSS while building an interactive and accessible app.
+
+### Prerequisites
+
+This guide assumes that you have a basic working knowledge of the following:
+
+- Tailwind CSS
+- TypeScript in React
+- building React UI components
+
+We will not be covering these topics in detail here.
+
+The end result of this guide is an interactive media player interface.
+Here's what it will look like in the end:
+
+<img src="/static/images/base-with-tailwind-css/player-final.png" alt="Screenshot of the media player used as example in the guide, designed by the Tailwind Labs team" style="width: 796px; margin-top: 8px; margin-bottom: 8px;" />
```

Comment:
A note for the future, as an effort to be able to more easily share the global namespace for our CDN, I would propose that we move forward by using `/static/base/` as a prefix for the MUI Base product (same for the other products).

It's what we have started to do for MUI X https://github.com/mui/mui-x/tree/master/docs/public/static/x and MUI Toolpad. So here, something like this would help:

```suggestion
<img src="/static/base/with-tailwind-css/player-final.png" alt="Screenshot of the media player used as example in the guide, designed by the Tailwind Labs team" style="width: 796px; margin-top: 8px; margin-bottom: 8px;" />
```

Reply:
On a different note, this image looks pixelized on my screen. Are we sure about the resolution? I would have expected 752 * 2 pixels = 1504 wide but we have 1280.

Reply:
On a different note, I first thought it was a "live demo", something that we run against our visual regression test suite. Would it make more sense to embed a codesandbox, like in https://mui.com/material-ui/getting-started/usage/#quick-start? Or at minimum to link one?